core/templates/dev/head/app.js

```diff
@@ -40,7 +40,7 @@ oppia.constant(
 oppia.constant('PLACEHOLDER_OUTCOME_DEST', '/');
 oppia.constant('INTERACTION_DISPLAY_MODE_INLINE', 'inline');
 oppia.constant('DEFAULT_RULE_NAME', 'Default');
-oppia.constant('CLASSIFIER_RULESPEC_STR', 'FuzzyMatches');
+oppia.constant('CLASSIFIER_RULE_STR', 'FuzzyMatches');
```

Comment:
I think this should be CLASSIFIER_RULE_TYPE or RULE_TYPE_CLASSIFIER (since that's what it is; the old name was pretty awful). This change also needs to be made in the backend in order to avoid confusion.